dis_snek/models/discord/guild.py

```diff
@@ -327,10 +327,36 @@ async def get_member(self, member_id: Snowflake_Type) -> Optional["models.Member
         """
         return await self._client.cache.fetch_member(self.id, member_id)
 
-    async def get_owner(self) -> "models.Member":
-        # maybe precache owner instead of using `get_owner`
+    def get_member(self, member_id: Snowflake_Type) -> Optional["models.Member"]:
+        """
+        Return the Member with the given discord ID.
+
+        Args:
+            member_id: The ID of the member:
+        Returns:
+            Member object or None
+        """
+        return self._client.cache.get_member(self.id, member_id)
+
+    async def fetch_owner(self) -> "models.Member":
+        """
+        Return the Guild owner
+
+        Returns:
+            Member object or None
+        """
+        # TODO: maybe precache owner instead of using `fetch_owner`
```

Comment:
The owner user object is cached during startup. 

This method should still exist, just in case the user is no longer in the cache, for some reason, might be worth looking into protecting that cached object, but thats an aside